cmd/namespace.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
```

Comment:
do we have a standard copyright for code files in this repo?

Reply:
boilerplate removed, aligned with decK